test/client/integration.test.ts

```diff
@@ -26,6 +27,14 @@ import * as mcl from "../../ts/mcl";
  * and reach a correct end state.
  */
 describe("Client Integration", function() {
+    before(async () => {
+        await del("./leveldb");
```

Comment:
We should make the leveldb path configurable in future work, out of scope for this PR.